docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -279,12 +279,37 @@ policies have the prefix ``hip_``.
  CUDA/HIP Execution Policies               Works with    Brief description
  ========================================= ============= =======================================
  cuda/hip_exec<BLOCK_SIZE>                 forall,       Execute loop iterations
-                                           scan,         in a GPU kernel launched
-                                           sort          with given thread-block
-                                                         size. Note that the
+                                           scan,         directly mapped to global threads
```

Comment:
👍 I wish there were a good way to embedd a summary in a rst table so that we don't have to repeat the same description for multiple policies. Oh well.